uberon_edit.obo

```diff
@@ -195307,6 +195307,21 @@ intersection_of: connects UBERON:0002116 ! ileum
 relationship: part_of UBERON:0002108 ! small intestine
 created_by: http://orcid.org/0000-0002-2825-0621
 
+[Term]
+id: UBERON:8410021
+name: inguinal region skin
+def: "The skin covering the inguinal region between the abdomen and the thigh on either side of the pubic bone." [http://orcid.org/0000-0001-7505-5418, PMID:32327715, PMID:468709, WikipediaVersioned:Groin&oldid=998393002]
+synonym: "groin skin" EXACT []
+synonym: "inguinal skin" EXACT []
+synonym: "skin of groin" EXACT []
+synonym: "skin of inguinal region" EXACT []
+synonym: "zone of skin of groin" EXACT []
+synonym: "zone of skin of inguinal region" EXACT []
+intersection_of: UBERON:0000014 ! zone of skin
+intersection_of: part_of UBERON:0008337 ! inguinal part of abdomen
+property_value: added_for_HCA "" xsd:string
```

Comment:
Seems to be some issue with subset tagging here. Guide here:  https://ontology-development-kit.readthedocs.io/en/latest/NewSlimTerm.html

Reply:
@dosumis 
Thanks, there may be more like that... I made a note to fix any other. Now there's a conflict...